list.txt

```diff
@@ -42,7 +41,6 @@
 079i080nhj.info
 080mail.com
 0815.ru
-0815.ry
```

Comment:
Hello!

Thanks for your PR, I do not understand why this is removed, please explain :)

Reply:
> Hello!
> 
> Thanks for your PR, I do not understand why this is removed, please explain :)

.ry TLD is not exist